javascript/javascript2/week3/fastest_presser.js

```diff
@@ -0,0 +1,43 @@
+const form = document.getElementById('game-start-form')
+const sCountNumber = document.getElementById('s-count')
+const lCountNumber = document.getElementById('l-count')
+
+let sCount = 0
+let lCount = 0
+const handleKeyDown = (event) => {
+  if (event.code === 'KeyL') {
+    lCount++
+    lCountNumber.innerHTML = lCount
+  }
+
+  if (event.code === 'KeyS') {
+    sCount++
+    sCountNumber.innerHTML = sCount
+  }
+}
+const startTimer = (duration) => {
+  document.addEventListener('keypress', handleKeyDown)
+
+  setTimeout(() => {
+    document.removeEventListener('keypress', handleKeyDown)
+    if (sCount > lCount) {
+      sCountNumber.innerHTML = 'WINNER'
+    } else if (lCount > sCount) {
+      lCountNumber.innerHTML = 'WINNER'
+    } else {
+      sCountNumber.innerHTML = 'TIE'
+      lCountNumber.innerHTML = 'TIE'
+    }
+  }, duration * 1000)
```

Comment:
I really recommend you to use a timer to make your app to a bit fun, so the use can see how many minute or seconds remains when He or She is playing.